pgx-version-updater/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "pgx-version-updater"
+version = "0.0.0"
+edition = "2021"
+authors = ["ZomboDB, LLC <zombodb@gmail.com>"]
+license = "MIT"
+description = "Standalone tool to update PGX Cargo.toml versions and dependencies in preparation for a release."
+homepage = "https://github.com/zombodb/pgx"
+repository = "https://github.com/zombodb/pgx"
+documentation = "https://docs.rs/pgx-utils"
+readme = "README.md"
+rust-version = "1.58"
```

Comment:
I think this `rust-version` line should be removed, pgx [doesn't have a MSRV](https://github.com/tcdi/pgx/pull/718).

Reply:
Right, this was a copy/paste issue!